src/map/skills/novice/skill_factory_novice.cpp

```diff
@@ -3,13 +3,23 @@
 
 #include "skill_factory_novice.hpp"
 
+#include "basic.hpp"
+#include "firstaid.hpp"
+#include "trickdead.hpp"
+#include "callbaby.hpp"
+
 std::unique_ptr<const SkillImpl> SkillFactoryNovice::create(const e_skill skill_id) const {
-#if 0
 	switch( skill_id ){
+		case NV_BASIC:
+			return std::make_unique<SkillBasic>(skill_id);
+		case NV_FIRSTAID:
+			return std::make_unique<SkillFirstAid>(skill_id);
+		case NV_TRICKDEAD:
+			return std::make_unique<SkillTrickDead>(skill_id);
+		case WE_CALLBABY:
+			return std::make_unique<SkillCallBaby>(skill_id);
+
```

Comment:
Notice how we are creating the instances here: https://github.com/rathena/rathena/pull/9516/files#diff-57eb9662dceda1152a51c5c4fb197dfa92830ff06fe1fc72206a577d2329172fR15-R26.

The concrete skill impl doesn't take in the ID, just the abstraction